content/learn/using-validated-pattern-operator.md

```diff
@@ -53,3 +53,62 @@ Review the rest of the form fields and check if they require changes. For first
 From there OpenShift GitOps will install the rest of the assets and artifacts for this pattern. Make sure to change your project to `All Projects` so you will see the other operators installing. E.g. Advanced Cluster Management (ACM).
 
 Please follow any other post-install instructions for the pattern on that pattern's `Getting started` page.
+
+## Using the in-cluster Gitea server
+
+The operator supports the ability to host a forked version of the pattern inside the cluster. This means you don't need to fork the pattern repository before deploying it in your cluster to be able to tweak while you work on it. This can be useful if you don't want to host the forked repository outside the cluster, or for a quick demo.
```

Comment:
```suggestion
The operator supports the ability to host a forked version of the pattern inside the cluster. This means you do not need to fork the pattern repository before deploying it in your cluster to be able to tweak while you work on it. This can be useful if you do not want to host the forked repository outside the cluster, or for a quick demo.
```